go/adbc/driver/snowflake/record_reader.go

```diff
@@ -36,6 +37,19 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const defaultConcurrentPrefetch = 10
+
+var concurrentPrefetch int
+
+func init() {
+	concurrentPrefetch = defaultConcurrentPrefetch
+	if value := os.Getenv("ADBC_SNOWFLAKE_DEFAULT_CONCURRENT_PREFETCH"); value != "" {
```

Comment:
Is this something a user should be able to configure? Just asking, no opinion.

Reply:
I think so. In the snowflake python connector it is also an env var with a default, so that's why i went the environment variable route.